seeders/20201109013159-create-demo-user.js

```diff
@@ -1,25 +1,55 @@
 /* eslint-disable */
+const {ROLE_ADMIN, ROLE_USER} = require('../constants/user.constants')
+const password = require('../helpers/password')
+
+const configUsers = [
+  {
+    roleId : ROLE_ADMIN,
+    quantity: 10,
+    firstName: 'User',
+    lastName: 'Admin',
+    email: 'admin',
+  },
+  {
+    roleId : ROLE_USER,
+    quantity: 10,
+    firstName: 'User',
+    lastName: 'Regular',
+    email: 'user',
+  },
+]
+
+async function createDemoUsers (configUsers) {
+  let users = []
+  for await (const item of configUsers) {
```

Comment:
fer como comentario no mas, el **for await** para este caso alcanzaria con **for** solo, ya que el iterable que vas a recorrer configUSers, tiene items que no contienen codigo asincrono.